cypress/integration/MultipleReferenceEditor.spec.ts

```diff
@@ -33,6 +33,6 @@ describe('Multiple Reference Editor', () => {
     openPage();
     findLinkExistingBtn().click();
     findLinkExistingBtn().click(); // inserts 2 cards
-    findLinkExistingBtn().should('not.be.visible'); // limit reached, button hidden.
+    findLinkExistingBtn().should('not.exist'); // limit reached, button hidden.
```

Comment:
Use `not.exist` when asserting an element doesn't exist in the DOM. Fails otherwise in Cypress > v6